src/ActionsImporter/App.cs

```diff
@@ -16,15 +15,19 @@ public class App
     public bool IsPrerelease { get; set; }
     public bool NoHostNetwork { get; set; }
 
+    private readonly ImmutableDictionary<string, string> EnvironmentVariables;
```

Comment:
```suggestion
    private readonly ImmutableDictionary<string, string> _environmentVariables;
```

nit: it's convention to use _camelCase for private fields and PascalCase for public/protected properties.